lib/dashdb.js

```diff
@@ -113,9 +124,9 @@ DASHDB.prototype.createTable = function(model, cb) {
                         'currently supported')));
   } else {
     tasks.push(function(callback) {
-      var sql = 'CREATE TABLE ' + tableSchema + '.' + tableName +
-          ' (' + columnDefinitions + ') ORGANIZE BY ROW;';
-      self.execute(sql, callback);
+      var sql = new ParameterizedSQL('CREATE TABLE ' + tableSchema + '.' +
+        tableName + ' (' + columnDefinitions + ') ORGANIZE BY ROW;');
+      self.execute(sql.sql, callback);
```

Comment:
If you're calling the base connector's `execute` function here, I think you can do `self.execute(sql.sql, sql.params, opts, callback)` here.